### ISSUE TITLE
Deleted data from directus remains in cache

### ISSUE THREAD

**Issue by DieAlchemistenAG** [iKonrad/gatsby-source-directus#6]
Hello,

first of all, thanks for writing this plugin!

But I have an issue:
Queried Items remain in cache.
When fetching data, the plugin shows the correct amount of items, but the ones that i deleted after last query remain in the results.
The only way I was able to delete those entries is deleting the .cache folder in the root

Any ideas?

**Comment by heho** [comment]
yeah that is a problem at the moment. I plan to rewrite this plugin once Directus 7 hits the shelves, since I use it for my own blog and I think it will be necessary with the new directus API. Till then I suppose you will have to delete the cache folder. You could also try to implement that yourself I think you should be able to understand how to by looking at the plugin for wordpress.
